package.json

```diff
@@ -8,10 +8,10 @@
     "lint": "eslint . --fix",
     "list-sources": "node src/shared/sources/_lib/list-sources.js",
     "todos": "node tools/list-dev-todos.js",
-    "test": "tape 'tests/**/*-test.js' | node tests/save-test-results.js | node tools/clean-tap-output.js | tap-spec; node tests/warnings.js",
-    "test:reports": "node tests/run-report-tests.js | node tests/save-test-results.js | node tools/clean-tap-output.js | tap-spec; node tests/warnings.js",
-    "test:unit": "tape 'tests/unit/**/*-test.js' | node tests/save-test-results.js | node tools/clean-tap-output.js | tap-spec; node tests/warnings.js",
-    "test:integration": "tape 'tests/integration/**/*-test.js' | node tests/save-test-results.js | node tools/clean-tap-output.js | tap-spec; node tests/warnings.js",
+    "test": "arc hydrate && tape 'tests/**/*-test.js' | node tests/save-test-results.js | node tools/clean-tap-output.js | tap-spec && node tests/warnings.js",
```

Comment:
The second `&&` (before `node tests/warnings.js`) I added means that test failures from the preceding command cause the test suite to fail. (At least assuming that `tap-spec` returns 0 if and only if the tests pass, which it seems to do.) Without this change, test failures can slip through. For example, GitHub's CI reports a successful test for this earlier PR, but we can see that it actually failed:

https://github.com/covidatlas/li/runs/996045825?check_suite_focus=true#step:7:14